File: src/main/scala/io/izzel/taboolib/module/inject/Releasable.java
package io.izzel.taboolib.module.inject;

import org.bukkit.entity.Player;

/**
 * TabooLib
 * io.izzel.taboolib.module.inject.Releaseable
 *
 * @author bkm016
 * @since 2020/11/17 1:15 上午
 */
public interface Releasable {

    void release(Player player, String namespace);

}

File: common-util/src/main/kotlin/taboolib/common/platform/Releasable.kt
package taboolib.common.platform

/**
 * 关服自动释放，需要搭配 @Awake 使用，有点鸡肋。
 *
 * @author sky
 * @since 2021/6/24 5:03 下午
 */
interface Releasable {

    fun release()
}